CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+chore: add a volume to the dockerfile
```

Comment:
```suggestion
chore: Add a `tmp` volume to the docker file so that we can cache the proguard mapper since it expects to be passed a cached file.
```